cloudformation/aqua-ecs-ec2/README.md

```diff
@@ -23,6 +23,7 @@ A CloudFormation template is used to deploy Aqua Enterprise. This can be done ei
 
  - An ECS cluster with at least 2 instances registered
  - A VPC with at least 2 subnets
+ - A certificate via AWS Certificate Manager
```

Comment:
if we giving a yml with a none ssl, I think that we need to update the guide to something like this:
A certificate via AWS Certificate Manager or use the yml without the ssl. wdyt?